src/radical/pilot/task_overlay/worker.py

```diff
@@ -217,6 +217,29 @@ def _call(self, data):
         return out, err, ret
 
 
+    # --------------------------------------------------------------------------
+    #
+    def _mpi(self, data):
+        '''
+        launch an rp.MPI_LAUNCH_METHOD for the task
+        '''
+
+        try:
+            cmd = rp.agent.launch_method.construct_command(data,
```

Comment:
Instead of using the whole package, why not just import what is needed?

Based on the linters, `rp` does not exist, creating a bug.

Reply:
> I would like to see a test for this or at least why it cannot be tested. Also, the linters are showing an important bug.

It can't be tested as it does not work yet.  I converted the PR into a comment for now.  This is really just about closing that branch for the time being as we won't have development on this for a while...

Reply:
Okay, then let's move forward.